release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -1224,6 +1385,56 @@ Previously, pre-flight checks did not account for {rh-openstack-first} resource
+Consequently, user-defined rules would not be evaluated until the node came back online.
++
+With this release, the Cluster Monitoring Operator (CMO) now configures hard anti-affinity rules to ensure that the two Thanos Ruler pods are scheduled on different nodes. 
+As a result, a single-node outage no longer creates a gap in user-defined rule evaluation. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1955490[*BZ#1955490*])
+
+* Previously, the Prometheus service would become unavailable when the two Prometheus pods were located on the same node and that node experienced an outage.
+This situation occurred because the Prometheus pods had only soft anti-affinity rules regarding node placement.
+Consequently, metrics would not be collected, and rules would not be evaluated until the node came back online.
++
+With this release, the Cluster Monitoring Operator configures hard anti-affinity rules to ensure that the two Prometheus pods are scheduled on different nodes.
+As a result, Prometheus pods are now scheduled on different nodes, and a single node outage no longer creates a gap in monitoring.(link:https://bugzilla.redhat.com/show_bug.cgi?id=1949262[*BZ#1949262*])
+
+* Previously, during {product-title} patch upgrades, the Alertmanager service might become unavailable because either the three Alertmanager pods were located on the same node or the nodes running the Alertmanager pods happened to reboot at the same time. 
+This situation was possible because the Alertmanager pods had soft anti-affinity rules regarding node placement and no pod disruption budget. 
+This release enables hard anti-affinity rules and pod disruption budgets to ensure no downtime during patch upgrades for the Alertmanager and other monitoring components. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1955489[*BZ#1955489*])
+
+* Previously, a false positive `NodeFilesystemSpaceFillingUp` alert was triggered when the file system space was occupied by many Docker images. 
+For this release, the threshold to fire the `NodeFilesystemSpaceFillingUp` warning alert is now reduced to 20% space available, instead of 40%, which stops the false positive alert from firing. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1987263[*BZ#1987263*])
+
+* Previously, alerts for the Prometheus Operator component did not apply to the Prometheus Operator that runs the `openshift-user-workload-monitoring` namespace when user-defined monitoring is enabled.
+Consequently, no alerts fired when the Prometheus Operator that manages the `openshift-user-workload-monitoring` namespace encountered issues.
++
+With this release, alerts have been modified to monitor both the `openshift-monitoring` and `openshift-user-workload-monitoring` namespaces.
+As a result, cluster administrators receive alert notifications when the Prometheus Operator that manages user-defined monitoring encounters issues. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2001566[*BZ#2001566*])
+
```

Comment:
same for bug 2001566, we have mentioned in the Changed section
`Extended PrometheusOperator alerts to include the Prometheus operator running in the openshift-user-workload-monitoring namespace.`

Reply:
@simonpasquier Should we delete this bug fix because we mention it in the Changed section for Alerts?

Reply:
keeping the section per Simon's previous comment